apps/web/components/Organization/OfficeCard/OfficeCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Box } from '@island.is/island-ui/core'
+import * as styles from './OfficeCard.treat'
+
+interface OfficeCardProps {
+  name: string
+  address: string
+  city: string
+  openingHours: string
+}
+
+export const OfficeCard: React.FC<OfficeCardProps> = ({
+  name,
+  address,
+  city,
+  openingHours,
+}) => {
+  return (
+    <>
+      <Box
+        borderRadius="large"
+        overflow="hidden"
+        background="blue100"
+        boxShadow="subtle"
+        border="standard"
+        borderColor="blue200"
+        marginBottom={[1, 1, 1]}
+        marginTop={[1, 1, 1]}
+        paddingX={[3, 3, 3]}
+        paddingY={[3, 3, 3]}
+      >
+        <h1 className={styles.title}>{name}</h1>
+        <p style={{ fontWeight: 300, marginTop: 14 }}>{address}</p>
```

Comment:
We should try to avoid inline styles and use treat for styling if necessary. It looks like this could be solved using the default variant from `Text`. 
```suggestion
        <Text marginTop={1}>{address}</Text>
```
Regarding the spacing we should try to limit spacings to the defined spacing variables, currently they are mostly a multiple of 8, but I can see you are using the paragraph spacings defined in figma, so maybe that's something we should add to the defined list of spacings.

Reply:
I've added paragraph spacing to the theme so when https://github.com/island-is/island.is/pull/2705 will be merged this could be
```suggestion
        <Text marginTop='p3'>{address}</Text>
```